gallery/quick_start_with_transforms_v2.py

```diff
@@ -0,0 +1,96 @@
+"""
+==================================
+Quick-start with Transforms API v2
+==================================
+
+This example illustrates how to update existing user's data augmentations pipeline with
+new Torchvision Transforms API v2 (:ref:`image transformations <transforms>`).
+
+"""
+
+#######################################
+# Classification pipeline
+# -----------------------
+# For the classification task, let us take as an example
+# training data augmentation pipeline for the ImageNet (`classification reference script <https://github.com/pytorch/vision/tree/96dbada4d588cabbd24ab1eee57cd261c9b93d20/references/classification>`_).
+# The only change we need is to replace the imported module `transforms` by `v2`:
+
+#######################################
+#
+#   .. code-block:: diff
+#
+#       - from torchvision.transforms import transforms as T
```

Comment:
Should we change this to the exact old import?

https://github.com/pytorch/vision/blob/e0eed2cb6629f36fc27414b7c5cb29cf0c338909/references/classification/presets.py#L2

Because otherwise the removed `autoaugment` is undefined.

Reply:
On the other hand, the snippet below is not exactly in our code like that anyway, right? So maybe we just fix the old line to say `T.AutoAugment(policy=aa_policy, interpolation=mode),` and thus showcase that there is no change whatsoever by using v2.